extensions/vscode/build.mjs

```diff
@@ -1,6 +1,6 @@
 import * as esbuild from 'esbuild';
-import fs from 'node:fs';
 import { execa } from 'execa';
+import fs from 'node:fs';
```

Comment:
I think we should enforce this with
- https://github.com/IanVS/prettier-plugin-sort-imports
- or https://github.com/un-ts/eslint-plugin-import-x/blob/master/docs/rules/order.md.

Reply:
Yea, I always use the VSC organize imports feature to order the imports which works really well and I like it a lot. Wondering if those plugins do the same or if VCS does something different. But generally I like that idea a lot as well cause I always order my imports.

Reply:
Love to enforce this too for better consistency across different machines and developers.

Reply:
Also to avoid constant reorderings of imports in PRs.